https://github.com/openldap/openldap/commit/8e3f87f86a51e78bffefb85968e5684213422cb7

From: Orgad Shaneh <orgad.shaneh@audiocodes.com>
Date: Tue, 25 Jan 2022 17:38:46 +0200
Subject: [PATCH] ITS#9788 Fix make jobserver warnings

Running make -j8 issues the following warning for each directory with
make 4.3:
make[2]: warning: -j8 forced in submake: resetting jobserver mode.

There is no need to pass MFLAGS. Make picks it up from the
environment anyway.
--- a/build/dir.mk
+++ b/build/dir.mk
@@ -21,7 +21,7 @@ all-common: FORCE
 	@echo "Making all in `$(PWD)`"
 	@for i in $(SUBDIRS) $(ALLDIRS); do 		\
 		echo "  Entering subdirectory $$i";		\
-		( cd $$i && $(MAKE) $(MFLAGS) all );		\
+		( cd $$i && $(MAKE) all );		\
 		if test $$? != 0 ; then exit 1; fi ;	\
 		echo " ";								\
 	done
@@ -30,7 +30,7 @@ install-common: FORCE
 	@echo "Making install in `$(PWD)`"
 	@for i in $(SUBDIRS) $(INSTALLDIRS); do 	\
 		echo "  Entering subdirectory $$i";		\
-		( cd $$i && $(MAKE) $(MFLAGS) install );	\
+		( cd $$i && $(MAKE) install );	\
 		if test $$? != 0 ; then exit 1; fi ;	\
 		echo " ";								\
 	done
@@ -39,7 +39,7 @@ clean-common: FORCE
 	@echo "Making clean in `$(PWD)`"
 	@for i in $(SUBDIRS) $(CLEANDIRS); do		\
 		echo "  Entering subdirectory $$i";		\
-		( cd $$i && $(MAKE) $(MFLAGS) clean );	\
+		( cd $$i && $(MAKE) clean );	\
 		if test $$? != 0 ; then exit 1; fi ;	\
 		echo " ";								\
 	done
@@ -48,7 +48,7 @@ veryclean-common: FORCE
 	@echo "Making veryclean in `$(PWD)`"
 	@for i in $(SUBDIRS) $(CLEANDIRS); do		\
 		echo "  Entering subdirectory $$i";		\
-		( cd $$i && $(MAKE) $(MFLAGS) veryclean );	\
+		( cd $$i && $(MAKE) veryclean );	\
 		if test $$? != 0 ; then exit 1; fi ;	\
 		echo " ";								\
 	done
@@ -57,7 +57,7 @@ depend-common: FORCE
 	@echo "Making depend in `$(PWD)`"
 	@for i in $(SUBDIRS) $(DEPENDDIRS); do		\
 		echo "  Entering subdirectory $$i";		\
-		( cd $$i && $(MAKE) $(MFLAGS) depend );	\
+		( cd $$i && $(MAKE) depend );	\
 		if test $$? != 0 ; then exit 1; fi ;	\
 		echo " ";								\
 	done
